languages/en.ini

```diff
@@ -356,6 +356,8 @@ edit_list_fail = "Sorry, you are not permitted to edit this list"
 edit_list_success = "List successfully updated."
 Edition = "Edition"
 EDS Results = "EDS Results"
+eds_databases = "Search Relevant Databases"
+eds_databases_intro = "Search directly in databases for enhanced functionality."
```

Comment:
This suggested language comes from an instruction librarian colleague.  But discussion welcome.  I asked her to consider how to concisely explain to patrons something that's not at all obvious:

- This is not a facet to narrow down these results, like the list of sources on the Articles tab.  This will actually open up the database.
- That said, the list of databases is still drawn from the EDS results to the left.
- Given that in theory all content of those databases is indexed by EDS and available here via the API, what (concisely) is the reason someone should exit to the database UI?